pkg/operator/config_reloader_test_lib.go

```diff
@@ -210,15 +211,11 @@ func TestSidecarsResources(t *testing.T, makeStatefulSet func(reloaderConfig Con
 			for _, c := range sset.Spec.Template.Spec.Containers {
 				if strings.HasSuffix(c.Name, "config-reloader") {
 					foundContainer = true
-				}
-				if strings.HasSuffix(c.Name, "config-reloader") && !reflect.DeepEqual(c.Resources, tc.expectedResources) {
-					t.Fatalf("Expected resource requests/limits:\n\n%s\n\nGot:\n\n%s", tc.expectedResources.String(), c.Resources.String())
```

Comment:
Am I missing something or does the `strings.HasSuffix` disappear with your change? Was that intentional?

Reply:
The ```string.HasSuffix``` is already checked in the above statement, so I thought I can add the ```t.Fatalf```(originally line 215) inside that if block (```require.True``` statement from line 214).

Reply:
I just needed to check the second condition i.e ```reflect.DeepEqual```

Reply:
Ah ok, got it! Good catch

One last suggestion: we could use `require.Equal()` instead of `require.True(... reflect.DeepEqual(...))`